lib/process-services-cloud/src/lib/task/task-header/mocks/task-details-cloud.mock.ts

```diff
@@ -26,7 +27,7 @@ export const assignedTaskDetailsCloudMock = new TaskDetailsCloudModel(
         'name': 'This is a new task ',
         'description': 'This is the description ',
         'createdDate': 1545048055900,
-        'dueDate': 1545091200000,
+        'dueDate': moment(new Date()).valueOf(),
```

Comment:
You are wrapping the date and then unwrapping it back, use `Date.now()` to get the number
```suggestion
        'dueDate': Date.now(),
```